src/Illuminate/Mail/Transport/MandrillTransport.php

```diff
@@ -45,7 +45,7 @@ public function send(Swift_Mime_Message $message, &$failedRecipients = null)
             'key' => $this->key,
             'to' => $this->getToAddresses($message),
             'raw_message' => $message->toString(),
-            'async' => false,
+            'async' => true,
```

Comment:
this fix here is important to allow mandrill to queue them for later if someone's gone over their hourly limit, rather than it just failing